.github/workflows/build.yml

```diff
@@ -0,0 +1,18 @@
+on: [push, pull_request]
+name: Build
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.15.x]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
I had comments in #61 about adjusting these, it would be good if we use the same matrix in both but since #62 is a superset I thought we could use that to further discuss.

Reply:
Thanks. I've updated the systems this workflow will run on based on your comment.